shared/src/commonMain/kotlin/com/intive/picover/shared/licenses/LicensesScreen.kt

```diff
@@ -2,4 +2,4 @@ package com.intive.picover.shared.licenses
 
 import cafe.adriel.voyager.core.screen.Screen
 
-internal expect class LicensesScreen : Screen
+internal expect class LicensesScreen() : Screen
```

Comment:
Fixes error `Expected class LicensesScreen does not have default constructor` on call site in